### DISCUSSION TITLE
Memory retained after screen is removed from stack.

### DISCUSSION THREAD

**Discussion by akshaya-rao-s** [react-navigation/react-navigation#11314] [Q&A]
I have Screens A and B. A has a big image. from A I use relace action and replace it with screen B. Not if i see the android profiler, the big image is still in memory. Feels like a memory leak to me. And if I keep doing replace actions between two empty screens, the memory usage keeps increasing slowly.  Any solution or ideas?

**Comment by svpease** [comment]
Did you ever find a solution for this?

I'm currently trying to pinpoint in the code where the actual management of a screen's memory is so if I have to I can just fork and monkeypatch a fix, but there's a lot of rabbitholes to chase..
